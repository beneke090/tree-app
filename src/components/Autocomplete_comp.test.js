import React from 'react';
import renderer from 'react-test-renderer';
import AutocompleteComp from './Autocomplete_comp';

describe('AutocompleteComp', () => {
  test('should match snapshot.', () => {
    const component = renderer.create(
      <AutocompleteComp
        label="testlabel"
        items={['a', 'b', 'c']}
        onChange={() => {}}
      />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
