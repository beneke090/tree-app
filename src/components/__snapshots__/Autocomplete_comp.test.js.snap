// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AutocompleteComp should match snapshot. 1`] = `
<label
  aria-expanded={false}
  aria-haspopup="listbox"
  aria-labelledby="use-downshift-0-label"
  aria-owns={null}
  className="block mt-4"
  role="combobox"
>
  <span
    className="text-gray-700"
    htmlFor="use-downshift-0-input"
    id="use-downshift-0-label"
  >
    testlabel
  </span>
  <br />
  <input
    aria-activedescendant={null}
    aria-autocomplete="list"
    aria-controls={null}
    aria-labelledby="use-downshift-0-label"
    autoComplete="off"
    className="form-input mt-1 block w-half inline-block"
    id="use-downshift-0-input"
    onBlur={[Function]}
    onChange={[Function]}
    onKeyDown={[Function]}
    placeholder="search..."
    value=""
  />
  
</label>
`;
